### PR TITLE
Hive e2e 4.9

### PR DESCRIPTION
**Pull request by 2uasimojo**
Add a new e2e job that runs on a 4.9 hub cluster.

### CODE REVIEW

File: ci-operator/config/openshift/hive/openshift-hive-master.yaml [hunk 1]
```diff
@@ -106,12 +114,31 @@ tests:
...
+    test:
+    - as: test
+      cli: latest
+      commands: CLOUD=aws make test-e2e
+      dependencies:
+      - env: HIVE_IMAGE
+        name: hive
+      - env: RELEASE_IMAGE
+        name: release:e2e-spoke
+      from: src
```
Comment (2uasimojo) [inline ci-operator/config/openshift/hive/openshift-hive-master.yaml:137]: Is this `from` right?

**Comment by 2uasimojo** [comment]
@stevekuznetsov As discussed -- would you please have a look at this?

**Comment by 2uasimojo** [comment]
/test pj-rehearse

e2e-pool should have succeeded. I think this was a flake, but let's make sure.

File: ci-operator/config/openshift/hive/openshift-hive-master.yaml [hunk 2]
```diff
@@ -106,12 +119,31 @@ tests:
...
           cpu: 100m
     workflow: generic-claim
+- as: e2e-4.9
+  steps:
+    cluster_profile: aws
+    dependencies:
+      RELEASE_IMAGE_LATEST: release:e2e-hub-4.9
+    test:
+    - as: test
+      cli: latest
```
Comment (2uasimojo) [inline ci-operator/config/openshift/hive/openshift-hive-master.yaml:135]: This (I think) seems to be [looking for](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/21007/rehearse-21007-pull-ci-openshift-hive-master-e2e-4.9/1425528272367652864#1:build-log.txt%3A48) an image called `stable-e2e-hub-4.9`. Why?

Comment (stevekuznetsov) [reply ci-operator/config/openshift/hive/openshift-hive-master.yaml:135]: The `cli` here is trying to import the `cli` image from the latest release. You likely want

```yaml
cli: e2e-spoke
```

File: ci-operator/config/openshift/hive/openshift-hive-master.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply ci-operator/config/openshift/hive/openshift-hive-master.yaml:142]: This chooses what image runs your tests. So, you clone the source then run `CLOUD=aws make test-e2e` - only you know if that's right

File: ci-operator/config/openshift/hive/openshift-hive-master.yaml (hunk 2 above)
Comment (2uasimojo) [reply ci-operator/config/openshift/hive/openshift-hive-master.yaml:135]: /me tries...

File: ci-operator/config/openshift/hive/openshift-hive-master.yaml (hunk 1 above)
Comment (2uasimojo) [reply ci-operator/config/openshift/hive/openshift-hive-master.yaml:142]: Cool, thanks. Yes, I believe that's what we want.

**Comment by 2uasimojo** [comment]
Blocked on https://issues.redhat.com/browse/DPTP-2448

**Comment by 2uasimojo** [comment]
https://github.com/openshift/ci-tools/pull/2372 merged (not sure if it needs to be percolated into ci-operator though).

**Comment by 2uasimojo** [comment]
@stevekuznetsov This appears to continue to suffer the [same error](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/21007/rehearse-21007-pull-ci-openshift-hive-master-e2e-4.9/1440323838503030784#1:build-log.txt%3A37). What's next?

**Pull request closed without merging**